.travis.yml

```diff
@@ -18,7 +18,7 @@ before_script:
 
 script:
   - bundle exec rake spec
-  - bundle exec govuk-lint-ruby app spec
+  - bundle exec govuk-lint-ruby app spec --format clang
```

Comment:
Can we lint more?

 ```
- bundle exec govuk-lint-ruby app config db lib spec --format clang
 - bundle exec govuk-lint-sass app/assets/stylesheets
```

Reply:
`govuk-lint-sass` throws up many errors. Can we add this in a separate PR?

Reply:
100%. I'll do that this afternoon 👍